### DISCUSSION TITLE
Anyone doing cool things with their solo2 key?

### DISCUSSION THREAD

**Discussion by bbigras** [solokeys/solo2-cli#26] [General]
Maybe as an ssh-agent, with LUKS or HashiCorp vault.

**Comment by macxcool** [comment]
Just got mine, so not really yet. The only cool thing I've done is have it fail a firmware update on Arch Linux which made me fear I'd bricked it. Then I used Chocolatey to install Rust and gcc on Windows, compiled solo2-cli and ran it there. It worked! Phew. ;-)
I _did_ set it up with Pam so I can log in to my Linux box with it. Simple things...
